### PR TITLE
[docs] Update airgapped install

### PR DESCRIPTION
**Pull request by ddhodge**
#8311 updated URLs

### CODE REVIEW

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/install-software/airgapped.md [hunk 1]
```diff
@@ -65,13 +65,13 @@ $ cd /opt/downloads
...
 ```sh
-$ wget https://downloads.yugabyte.com/replicated.tar.gz
+$ wget --trust-server-names https://get.replicated.com/airgap
 ```
 
 Download the `yugaware` binary and change the following number, as required:
 
 ```sh
-$ wget https://downloads.yugabyte.com/yugaware-2.1.2.0-b10.airgap
+$ wget https://downloads.yugabyte.com/yugaware-2.4.4.0-b7.airgap
```
Comment (polarweasel) [inline docs/content/latest/yugabyte-platform/install-yugabyte-platform/install-software/airgapped.md:74]: Seems odd to be providing a `stable` release download in the `latest` docs. Should it be this instead?
```suggestion
$ wget https://downloads.yugabyte.com/yugaware-2.7.1.1-b1.airgap
```

Comment (polarweasel) [reply docs/content/latest/yugabyte-platform/install-yugabyte-platform/install-software/airgapped.md:74]: (And if so, you'll need to also update the same file in the stable folder)

Comment (ddhodge) [reply docs/content/latest/yugabyte-platform/install-yugabyte-platform/install-software/airgapped.md:74]: fixed

**Review by polarweasel — APPROVED** [review]
LGTM. Is there a good way for us to tell the reader _where_ they might find the version and build number they need, or not really? (download.yugabyte.com may be the easiest place to point them, if we do want to point them anywhere)

**Pull request merged**